### DISCUSSION TITLE
Built-In CSS Support SSR Issue

### DISCUSSION THREAD

**Discussion by epozsh** [vercel/next.js#13625] [Help] [answered]
Hello,

Global Stylesheet does not loading on SSR.

I am using custom server and custom document.

When i disable javasrcipt (for ssr loading).

The global styles does not loaded.

Is there any extra configuration to load Css on SSR?

Best Regards

**Comment by rafaelalmeidatk** [comment, marked as answer]
This is the expected behavior with the development environment. The production build will include the CSS in SSR so you can see the styles with JavaScript disabled

**Reply by codemilli** [reply]
It is expected because of hmr right?
is there any way to prevent it in dev mode?

**Reply by rafaelalmeidatk** [reply]
> It is expected because of hmr right?

Yeah, exactly. 

> is there any way to prevent it in dev mode?


Nope, there is no way to turn off HMR.

**Reply by rafinskipg** [reply]
omg, is that explained somewhere? @rafaelalmeidatk  I stopped using CSS because this.
